ingestfile

```diff
@@ -283,7 +295,7 @@ fi
         done
     fi
 
-    if [ "${CROPDECISION}" = "" ] ; then
+    if [ "${CROPDECISION}" = "" ] && [ "${SILENT_RUNNING}" == "" ] ; then
```

Comment:
maybe this section could have a single `if not silent running` over the crop, formula, audiodecision, and cleanup checks, rather than integrating the logic into each check individually.